src/components/Avatar/avatarRules.ts

```diff
@@ -33,15 +33,20 @@ const getPresenceSpanLeft = (size: number, presenceIconPadding: number) => {
   )
 }
 
-const getPresenceSpanTop = (size: number, presenceIconPadding: number) => {
+const getPresenceSpanTop = (size: number, presenceIconPadding: number, src: string) => {
```

Comment:
Let's rename this method to be void of implementation (span).  That will prevent future naming "breaks" due to refactors.